e2e/clients.py

```diff
@@ -0,0 +1,76 @@
+"""
+Client fixtures module
+"""
+
+import subprocess
+import time
+import requests
+
+import pytest
+
+from kubernetes import client, config
+import kfp
+
+def client_from_config(cluster, region):
```

Comment:
It reads the cluster config from the k8s context. It's equivalent to:
```
kubectl config use-context <context>
```

i was leaving the door open for the multiple cluster, multiple concurrent test run case where the default context could be changing across runs otherwise.

Reply:
looks ok for now, can be reassesed later